### ISSUE TITLE
size mismatch for conv1.weight: copying a param with shape torch.Size([96, 1, 3, 3]) from checkpoint, the shape in current model is torch.Size([96, 3, 3, 3]).

### ISSUE THREAD

**Issue by ahmetfurkaann** [milesial/Pytorch-UNet#455]
I did not get any runtime errors while training the model, but in the validation part, all masks return with zero labels. In addition, when trying to use the model weight in predict.py this error comes up?

My images and masks are in .npy format. My images are in grayscale.
![image](https://github.com/milesial/Pytorch-UNet/assets/61904563/7242a103-f4f7-4fb1-b2d7-b93e141a644a)

**Comment by naguileraleal** [comment]
Hello!
I have the same issue. Did you manage to solve it @ahmetfurkaann ?
How are your input masks formatted? Do the files have a channel for each class, each with 1's or 255's? Or just two channels and a class index for each pixel?

**Comment by algwang** [comment]
> 你好！我有同样的问题。你设法解决了吗？您的输入掩码如何格式化？文件是否每个类都有一个频道，每个频道都有 1 或 255？或者每个像素只有两个通道和一个类索引？

Hello, have you solved the problem that the prediction results are all black? Looking forward to your answer
